lambdatimes/src/components/TopBar.js

```diff
@@ -1,23 +1,105 @@
 import React from 'react';
+import styled, {css} from 'styled-components'
 
 // Refactor this component to use styled components and not classNames. 
 // You can find the corresponding CSS in the CSS/index.css file
 
+const TopBarContainer = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: none;
+  flex-direction: row;
+  position: fixed;
+  height: 44px;
+  background-color: #333;
+`
+
+const Container = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: none;
+  align-items: none;
+  flex-direction: row;
+  color: #fff;
+  letter-spacing: 1px;
+  padding: 0 10px;
+  @media (min-width: 1280px) {
+    width: 1280px;
+  }
+`
+const ContainerLeft = styled.div`
+  display: flex;
+  justify-content: none;
+  align-items: center;
+  flex-direction: row;
+  flex: 1;
+  font-size: 11px;
+`
+
+const ContainerLeftSpan = styled.div`
+  cursor: pointer;
+  margin-right: 25%;
+  font-weight: bold;
+`
+
+const ContainerCenter = styled.span`
+  display: flex;
+  justify-content: none;
+  align-items: center;
+  flex-direction: row;
+  flex: 1;
+  font-size: 11px;
+`
+const ContainerCenterSpan = styled.span`
+  cursor: pointer;
+  margin-right: 5%;
+  ${ContainerCenterSpan}:hover {
+    text-decoration: underline;
+  }
+`
+
+const ContainerCenterSpanLast = styled.span`
+  cursor: pointer;
+  margin-right: 0;
+  ${ContainerCenterSpanLast}:hover {
+    text-decoration: underline;
```

Comment:
to add a pseudo-attribute like hover on styled components you can use the &:hover { <your styles go here> } in place of interpolation you used with ${ContainerCenterSpanLast}